### PR TITLE
Remove iot prefix from `iot_pkcs11`

### PR DESCRIPTION
**Pull request by lundinc2**
As part of migrating PKCS to a separate repository, the naming scheme is being updated to remove the `iot` prefix.

### CODE REVIEW

**Review by davwan01 — APPROVED** [review]
It seems this change aligns with the PR https://github.com/FreeRTOS/corePKCS11/pull/20.
Looks good to me.

**Comment by Sherryzhang2** [comment]
OK, got it. So this repo will be involved into the https://github.com/FreeRTOS/corePKCS11. Based on this information, this PR looks good to me.

**Review by pfalcon — APPROVED** [review]
Thanks @davwan01 and @Sherryzhang2 for providing additional context.

And thanks @lundinc2 for preparing this PR, I started to watch https://github.com/FreeRTOS/corePKCS11/pull/20 , and will merge this patch after the upstream change is merged.

**Comment by lundinc2** [comment]
Thanks for the review everyone. I have now merged the PR mentioned above.

**Comment by pfalcon** [comment]
Merged, thanks.

**Pull request merged**
